src/main/java/com/upsolver/datasources/jdbc/utils/SQLDrivers.java

```diff
@@ -0,0 +1,41 @@
+package com.upsolver.datasources.jdbc.utils;
+
+import java.io.IOException;
+import java.sql.DriverManager;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+public class SQLDrivers {
+    private final Collection<SQLDriver> drivers = new ArrayList<>();
+
+    public SQLDrivers() {
+        Collection<String> availableDrivers = DriverManager.drivers().map(driver -> driver.getClass().getName()).collect(Collectors.toSet());
+        Properties props = new Properties();
+        try {
+            props.load(getClass().getResourceAsStream("/drivers.properties"));
```

Comment:
don't you prefer just initializing a static list of "SqlDriver" in the code instead of parsing a .properties file and doing "split" and stuff on it?

Reply:
No, this way is better: it is easier to maintain.